backend/.pylintrc

```diff
@@ -505,7 +505,7 @@ known-third-party=enchant
 [DESIGN]
 
 # Maximum number of arguments for function / method.
-max-args=5
+max-args=7
```

Comment:
Because of [this line](https://github.com/Integreat/cms-django/blob/b3dc29f49535d143712115b394e1ad711d8a5e3b/backend/cms/views/push_notifications/push_notification_sender.py#L17), but probably it is better to leave the config as it is and just add an ignore-comment it in this line...

Reply:
I guess we should leave it with 5 args and as you said ignore the few excecptions